tripleo-ciscoaci/deployment/neutron_opflex/neutron-opflex-agent-container-puppet.yaml

```diff
@@ -0,0 +1,202 @@
+    properties:
+      ServiceData: {get_param: ServiceData}
+      ServiceNetMap: {get_param: ServiceNetMap}
+      DefaultPasswords: {get_param: DefaultPasswords}
+      EndpointMap: {get_param: EndpointMap}
+      RoleName: {get_param: RoleName}
+      RoleParameters: {get_param: RoleParameters}
+
+  # Merging role-specific parameters (RoleParameters) with the default parameters.
+  # RoleParameters will have the precedence over the default parameters.
+  RoleParametersValue:
+    type: OS::Heat::Value
+    properties:
+      type: json
+      value:
+        map_replace:
+          - map_replace:
+            - neutron::agents::ml2::ovs::bridge_mappings: NeutronBridgeMappings
+              #vswitch::ovs::enable_hw_offload: OvsHwOffload
+            - values: {get_param: [RoleParameters]}
+          - values:
+              NeutronBridgeMappings: {get_param: NeutronBridgeMappings}
+              #OvsHwOffload: {get_param: OvsHwOffload}
+
+  ContainersCommon:
+    type: /usr/share/openstack-tripleo-heat-templates/deployment/containers-common.yaml
+
+outputs:
+  role_data:
+    description: Role data for the Neutron OVS agent service.
+    value:
+      service_name: neutron_opflex_agent
+      config_settings:
+        map_merge:
+          - get_attr: [NeutronBase, role_data, config_settings]
+          - get_attr: [RoleParametersValue, value]
+          - neutron::agents::ml2::ovs::local_ip: {get_param: [ServiceNetMap, NeutronTenantNetwork]}
+      puppet_config:
+        config_volume: neutron_opflex
+        puppet_tags:
+        step_config: |
+          include ::tripleo::profile::base::ciscoaci_neutron_opflex
+        config_image: {get_param: ContainerNeutronOpflexAgentImage}
+        # We need to mount /run for puppet_config step. This is because
+        # puppet-vswitch runs the commands "ovs-vsctl list open_vswitch ."
+        # when running vswitch::ovs::enable_hw_offload: true
+        # ovs-vsctl talks to the ovsdb-server (hosting conf.db)
+        # on the unix domain socket - /run/openvswitch/db.sock
+        volumes:
+          - /lib/modules:/lib/modules:ro
+          - /run/openvswitch:/run/openvswitch
+          - /etc/os-net-config:/etc/os-net-config:ro
+      kolla_config:
+        /var/lib/kolla/config_files/ciscoaci_neutron_opflex_agent.json:
+          command: /bin/supervisord -c /etc/neutron/neutron_opflex_supervisord.conf
+          config_files:
+            - source: "/var/lib/kolla/config_files/src/*"
+              dest: "/"
+              merge: true
+              preserve_properties: true
+          permissions:
+            - path: /var/log/neutron
+              owner: neutron:neutron
+              recurse: true
+      docker_config_scripts: {get_attr: [ContainersCommon, container_config_scripts]}
+      docker_config:
+        step_4:
+          ciscoaci_neutron_opflex_agent:
+            start_order: 14
+            image: {get_param: ContainerNeutronOpflexAgentImage}
```

Comment:
Image name should be same as the one in  build_openstack_aci_containers.py script,  which is   DockerNeutronOpflexAgentImage.